.prettierrc

```diff
@@ -0,0 +1,5 @@
+{
+  "tabWidth": 2,
+  "semi": false,
+  "singleQuote": true
```

Comment:
[note]

I want to make sure this is okay.
Because both single quote and double quote are used.
I think single quote is more widely used in the repo and in general.
So I set `"singleQuote": true` here.

Reply:
yep also good. variation was due to different setups over the years i guess and not setting things up like this PR.